frontend/talentsearch/.env.example

```diff
@@ -2,9 +2,11 @@ APP_NAME="Talent Search"
 APP_ENV=local
 APP_KEY=
 APP_DEBUG=true
+TALENTSEARCH_APP_PROFILE_URL="http://localhost:8000/talent/profile"
```

Comment:
What you mean? Take it off?
May I add like this instead?
APPLICANTPROFILE_APP_DIR_URL="http://localhost:8000/applicant

Reply:
How this should going to be then?
APPLICANTPROFILE_APP_DIR="/talent/profile"
                  or
APPLICANTPROFILE_APP_DIR="/profile"

Reply:
In this commit you've added two new environment variables: `TALENTSEARCH_APP_PROFILE_URL` and `TALENTSEARCH_APP_PROFILE_DIR`.  But in `frontend/talentsearch/resources/js/talentSearchConstants.ts` I only see `TALENTSEARCH_APP_PROFILE_DIR` being used.  Is `TALENTSEARCH_APP_PROFILE_URL` being used somewhere else that I didn't notice?

Reply:
I could see some unexpected urls when I click the workpreferences link in storybook, it looks for `en/talent/preferences `instead of `en/talent/profile/preferences`

Reply:
Oh! I added because the existing TALENTSEARCH_APP_DIR has a corresponding URL so I just created one for TALENTSEARCH_PROFILE_APP_DIR.
I will delet it then.